semcore/d3-chart/__tests__/index.test.tsx

```diff
@@ -2525,6 +2527,8 @@ describe('d3 charts visual regression', () => {
       );
     };
 
-    expect(await snapshot(<Component />)).toMatchImageSnapshot();
+    expect(await snapshot(<Component />)).toMatchImageSnapshot({
+      failureThreshold: 0.0001,
```

Comment:
Do we really need it?

Reply:
Yes. Rendering circles in svg is not stable when compared by pixel. I added a clarifying comment